data/categories.txt

```diff
@@ -6321,7 +6328,7 @@ uk:^Магазин шин|магазин|крамниця
 vi:^Cửa hàng Lốp
 zh-Hans:^轮胎店|商店
 zh-Hant:^輪胎店|商店
-el:^Κατάστημα ελαστικών|κατάστημα
+el:^Βουλκανιζατέρ|κατάστημα ελαστικών|πωλήσεις
```

Comment:
If it is suitable, "πωλήσεις" probably can be moved from here to the synonym of the group term ```@shop```.

Reply:
ping on this comment